### DISCUSSION TITLE
Changes to nuget package lock include extra brackets and parentheses

### DISCUSSION THREAD

**Discussion by rbanks54** [renovatebot/renovate#18253] [Request Help] [answered]
I'm seeing the following in some recent PRs created by Renovate, when updating the nuget `packages.lock.json` file:

```
          "Azure.Identity": "[1.7.0, )",
          "Azure.Storage.Blobs": "[12.13.1, )",
          "CsvHelper": "[29.0.0, )",
          "Microsoft.Azure.Management.ResourceGraph": "[2.1.0, )",
          "Microsoft.Azure.OperationalInsights": "[1.1.0, )",
          "Microsoft.Extensions.Configuration.Binder": "[6.0.0, )",
          "Microsoft.Extensions.Configuration.CommandLine": "[6.0.0, )",
          "Microsoft.Extensions.Configuration.EnvironmentVariables": "[6.0.1, )",
          "Microsoft.Extensions.Configuration.Json": "[6.0.0, )",
          "Newtonsoft.Json": "[13.0.1, )",
          "System.Text.RegularExpressions": "[4.3.1, )"
```

I can't tell why it would be adding the extra characters, but I would suspect something in the match patterns is a little broken.

Is anyone else seeing this?

**Comment by viceice** [comment, marked as answer]
renovate isn't directly changing the lockfile, so it must be a recent dotnet cli change

**Comment by rbanks54** [comment]
Turns out I should just read the NuGet docs. 🤭

It's an ...unexpected?... syntax choice, but it's an open-ended version range specifier.

https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges
